docs/install/harvester-configuration.md

```diff
@@ -509,3 +509,33 @@ system_settings:
   http-proxy: '{"httpProxy": "http://my.proxy", "httpsProxy": "https://my.proxy", "noProxy": "some.internal.svc"}'
   ui-source: auto
 ```
+
+### `cluster_networks`
+
+#### Definition
+
+You can setup the default network in Harvester by configuring `cluster_networks`. Network configuration reference:
+
+- `vlan`: Setup for VLAN network. The following fields are supported by VLAN network:
+    - `enable` (bool, optional): enable VLAN network settings or not. Defaults to false.
+    - `description` (string, optional): Describe any comment or related information for ClusterNetworks. Defaults to "".
+    - `config` (dictionary, required): Specific type of `ClusterNetworks` configuration to be used. Valid configuration fields are:
+        - `defaultPhysicalNIC` (string, required): assign physical NIC to be external entry of VLAN network
```

Comment:
```suggestion
- `vlan`: Setup for VLAN network. The following fields are supported:
    - `enable`: enable VLAN network settings or not. Default value: `false`.
    - `description`: Additional information for `ClusterNetworks`. Default value: "".
    - `config`: `ClusterNetworks` configuration to be used. Valid configuration fields are:
        - `defaultPhysicalNIC` (string, required): assign a physical NIC to be external entry of VLAN network
```

Reply:
The original settings list makes a lot of sense with the type and mandatory/optional value. However as the whole page does not have anything like this, it's breaks the "style".

However, I will check with the team and come with an updated version were we do have these information for all fields.